pkg/scheduler/preemption/preemption.go

```diff
@@ -455,11 +457,15 @@ func fairPreemptions(preemptionCtx *preemptionCtx, candidates []*workload.Info,
 		logV.Info("Simulating fair preemption", "candidates", workload.References(candidates), "resourcesRequiringPreemption", preemptionCtx.frsNeedPreemption.UnsortedList(), "preemptingWorkload", klog.KObj(preemptionCtx.preemptor.Obj))
 	}
 
+	// DRS values must include incoming workload.
+	preemptionCtx.preemptorCQ.AddUsage(preemptionCtx.workloadUsage)
```

Comment:
non blocking suggestion to consider: 

Wdyt about the pattern like in `SimulateUsageRemoval`, we could have `SimulateUsageAddition`. Probably it does not help much in case of one workload, but helps to remember about reverting the change, and the name "simulate" suggests this is just temporary so it may be more suggestive to new readers of the code, wdyt?

Reply:
Nice suggestion. Done :)